### PR TITLE
Update library names

### LINKED ISSUE #691: libimobiledevice.dylib and others append version number
Follow-up of https://github.com/gluonhq/client-maven-plugin/issues/196

Recently, some of the libraries used by iosDeploy append the version number, like libusbmuxd.dylib to libusbmuxd-2.0.dylib and libimobiledevice.dylib to libimobiledevice-1.0.dylib. 

We need to give support to old ones too.

### PR DESCRIPTION
**Pull request by jperedadnr**
Fixes #691

### CODE REVIEW

File: src/main/java/com/gluonhq/substrate/util/ios/Deploy.java [hunk 1]
```diff
@@ -83,7 +84,7 @@ private void checkPrerequisites() throws IOException, InterruptedException {
         }
 
         // Check for libimobiledevice installed
-        List<String> libiPath = checkDependencyPaths("libimobiledevice");
+        List<String> libiPath = checkDependencyPaths(LIBIMOBILEDEVICE);
         if (libiPath == null || libiPath.isEmpty() || !Files.exists(Path.of(libiPath.get(0)))) {
             Logger.logSevere("Error finding libimobiledevice.dylib");
```
Comment (tiainen) [inline src/main/java/com/gluonhq/substrate/util/ios/Deploy.java:89]: Perhaps we should no longer mention .dylib here, but just inform the user that the library could not be found?

Comment (jperedadnr) [reply src/main/java/com/gluonhq/substrate/util/ios/Deploy.java]: In fact, this check is redundant, `checkDependencyPaths` does it too, and will throw already a RTE. I've done a small refactoring.

File: src/main/java/com/gluonhq/substrate/util/ios/Deploy.java [hunk 2]
```diff
@@ -220,24 +216,38 @@ private void copyAppToProducts(Path debugSymbolsPath, Path executablePath, Strin
...
+                .peek(libPath -> Logger.logDebug("lib " + nameLib + " found at " + libPath))
                 .collect(Collectors.toList());
+
+        if (list.isEmpty()) {
+            if (nameLib.contains("-")) {
+                Logger.logDebug("Trying old version of " + nameLib + ".dylib");
+                return getDependencyPaths(nameLib.split("-")[0]);
+            } else {
+                Logger.logSevere("Error: " + nameLib + ".dylib was not found");
+                throw new RuntimeException("Open a terminal and run the following command to install " + nameLib + ": \n\n" +
```
Comment (eugener) [inline src/main/java/com/gluonhq/substrate/util/ios/Deploy.java:246]: Codacy found an issue: [Avoid throwing raw exception types.](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=5814556)

**Comment by eugener** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity decreasing per file
==============================
+ src/main/java/com/gluonhq/substrate/util/ios/Deploy.java  -3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=5814556&bid=19036129)

**Comment by eugener** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity decreasing per file
==============================
+ src/main/java/com/gluonhq/substrate/util/ios/Deploy.java  -3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=5814556&bid=19036129)

**Comment by eugener** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity decreasing per file
==============================
+ src/main/java/com/gluonhq/substrate/util/ios/Deploy.java  -3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=5814556&bid=19036129)

**Pull request merged**